src/main/java/com/exactpro/epfast/template/xml/AbstractIdentity.java

```
@@ -20,9 +20,9 @@
 
 public abstract class AbstractIdentity implements Identity {
 
-    private String name;
+    private String name = "";
```

`name` is mandatory attribute and dosn't have any default for identity in xml if I'm not mistaken. When you add default for it you hide possible issue with provided xml.